### PR TITLE
Create khushal-sonawat.md

### PR DESCRIPTION
**Pull request by khushalsonawat**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit b9ce6f0496a4a18b1f5bced72f54c6e364996b1c at: https://66fe9c1d2515e4a1622ba608--meshery-docs-preview.netlify.app

File: docs/_meetings/week-of-09-30-2024/khushal-sonawat.md [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (vishalvivekm) [inline docs/_meetings/week-of-09-30-2024/khushal-sonawat.md:1]:
```suggestion
Name: Khushal Sonawat
```

Comment (khushalsonawat) [reply docs/_meetings/week-of-09-30-2024/khushal-sonawat.md:1]: Yes Vivek

**Comment by l5io** [comment]
🚀 Preview for commit 8dae5c6a023bf3ecfcfafcc730abddb9b7e6d6f6 at: https://66ff5065f506647f211b89e1--meshery-docs-preview.netlify.app

**Pull request merged**